### PR TITLE
Merge Dev to Main

### PR DESCRIPTION
**Pull request by Runicsaber**

### CODE REVIEW

File: app/product/[id]/details/3d/ArButton.tsx [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+  } from '@/components/ui/drawer';
+  import Image from 'next/image';
+  import { TbAugmentedReality } from 'react-icons/tb';
+  import Link from 'next/link';
+  import { Button } from 'antd';
+
+export default function ArButton({qrImage, objectID}: {qrImage: string | undefined, objectID: string}) {
+  return (
+    <>
+    { !isMobile ? //TODO make this use client
```
Comment (tatyanabortnik) [inline app/product/[id]/details/3d/ArButton.tsx:19]: does this still work?

Comment (Runicsaber) [reply app/product/[id]/details/3d/ArButton.tsx:19]: yes

**Review by Charbel-R — APPROVED** [review]
LGTM

**Review by tatyanabortnik — APPROVED** [review]
good

**Pull request merged**